addons/main/XEH_postInit.sqf

```diff
@@ -2,10 +2,35 @@
 
 [] call FUNC(arsenal_postInit);
 
+// Clear inventory of a container
+[QGVAR(clearAllCargo), {call FUNC(clearAllCargo)}] call CBA_fnc_addEventHandler;
+// Clear inventory of every backpack in a container
+[QGVAR(clearCargoBackpacks), {call FUNC(clearCargoBackpacks)}] call CBA_fnc_addEventHandler;
+
 if (isServer) then {
     [QGVAR(handleDisconnect), {
         addMissionEventHandler ["HandleDisconnect", FUNC(EHHandleDisconnect)];
     }] call CBA_fnc_addEventHandler;
+
+    [QGVAR(initHelper), {
+        params ["_ropeTop", "_holder"];
+        [{
+            params ["_ropeTop", "_pfhID"];
+            if (speed _ropeTop < 1 && {getPos _ropeTop # 2 < 1}) exitWith {
+                deleteVehicle _ropeTop;
+                [_pfhID] call CBA_fnc_removePerFrameHandler;
+            };
+        }, 1, _ropeTop] call CBA_fnc_addPerFrameHandler;
+        [{
+            params ["_holder", "_pfhID"];
+            if (speed _holder < 1 && {getPos _holder # 2 < 1}) exitWith {
+                private _pos = getPos _holder;
+                _pos set [2, 0];
+                _holder setPos _pos;
+                [_pfhID] call CBA_fnc_removePerFrameHandler;
+            };
+        }, 1, _holder] call CBA_fnc_addPerFrameHandler;
+    }] call CBA_fnc_addEventHandler;
```

Comment:
These pfhs handle 
1. deletion of _ropeTop, the helper object which is attached to the parachute, and from where the rope hangs, and 
2. repositioning of the weaponholdersimulated which holds the backpack, so that after it lands it isn't floating or half way in the ground

I tried using EPEContact eventhandlers, which seem to fire on player computers, but not on the server. Meanwhile these need to be monitored on the server in case of player disconnect.